alphagov/paas-admin
# [DO NOT MERGE] Make abbr keyboard accessible

**Needs a way to dismiss the "tooltip' otherwise it fails WCAG 1.4.13**

What
----

Make `abbr` element `title` property appear on keyboard focus and mouse hover, much like the browser native implementation but more accessible

How to review
-------------

- check out branch
- go to a page with an abbreviation (org, calculator)
- mouse hover and keyboard tab focus and evaluate the result


Who can review
---------------

not @kr8n3r
